package/agent/changelog.yml

```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 2.5.2
```

Comment:
I tend to just drop the pf-host-agent versioning here and replace it with a version number that is just for this package. As this package is not published from the point of view of Elastic Package Registry/Storage, replacing this version does not have a direct effect. In addition resources are missing to build the publishing pipeline for this package and so this repository might be closed and its content moved into another repository where the publishing pipeline is set up. In this case where the complete code for the package is moved to the new repository the process is similar to create a new package and the version number for the package does not matter anymore.

The downside I see is that there will be another version number.

Regardless - the version number that is used here can not be enforced on the installed version of the executable. If we would keep the version number of the package in sync with the version number of pf-host-agent it might be easier to name the installed version if issues are reported.

What do you think?

Reply:
I would keep the versions in sync to avoid confusion and help reporting of issues like you mentioned. So I guess it's OK to have this be `2.5.4` for now and we'll change it when we tag v3.x, which should be done next week (as `2.5.4` is incompatible with 8.8 stack release).